src/main/java/org/jenkinsci/plugins/junitrealtimetestreporter/RealtimeJUnitStep.java

```diff
@@ -132,6 +133,17 @@ public void setSkipMarkingBuildUnstable(boolean skipMarkingBuildUnstable) {
         this.skipMarkingBuildUnstable = skipMarkingBuildUnstable;
     }
 
+    public Long getParseInterval() {
+        return parseInterval;
+    }
+
+    @DataBoundSetter
+    public void setParseInterval(Long parseInterval) {
+        if (parseInterval == null || parseInterval.longValue() > 0) {
```

Comment:
this doesn't look right, if the value is null then set is? it's only ever going to set null here?

Reply:
We don't want a negative value (parse interval needs to be positive and higher than 0)
We allow null values (parse interval not set explicitely), then the current logic of "every 1/100 of a job estimated duration but not more often than every 5 seconds" will apply.

Actually I'm giving the option of setting a specific parse interval without changing the logic for existing builds. I personally find the current dynamic logic a bit weird, but unless we throw that to the bin and set a default value that is static, we need a way for the users to express they want the dynamic calculated parse rate, that is what the null value is.

But I'm open to suggestions if you think we can completely change the current behavior.

Reply:
I don’t understand your comment, the check was the wrong way round before from what I can tell but it seems fixed now :)